eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -9,6 +9,13 @@
          This version does not have source build intermediate.
          When a newer version is picked up, the intermediate should be utilized. -->
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.AnalyzerUtilities/*3.3.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Build/17.7.2" />
+    <UsagePattern IdentityGlob="Microsoft.Build.Framework/17.7.2" />
+    <UsagePattern IdentityGlob="Microsoft.NET.StringTools/17.7.2" />
+    <UsagePattern IdentityGlob="System.Configuration.ConfigurationManager/7.0*" />
+    <UsagePattern IdentityGlob="System.Diagnostics.EventLog/7.0*" />
+    <UsagePattern IdentityGlob="System.Reflection.MetadataLoadContext/7.0*" />
+    <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/7.0*" />
```

Comment:
In order for these to be added to the baseline, there needs to be corresponding entries for these packages in Version.Details.xml and Directory.Packages.props. This will allow source build to flow in the relevant version for these packages in the context of the VMR and for the repo to configuration to consume those versions.